scripts/testserver_codegen.sh

```diff
@@ -10,6 +10,5 @@ do
     autorest --input-file=./node_modules/@microsoft.azure/autorest.testserver/swagger/$i.json --output-folder=./test/integration/generated/$i --namespace=$i --use=.
     cd ./test/integration/generated/$i
     swift build
-    swift package generate-xcodeproj
```

Comment:
Why was this line removed?

Reply:
i don't want to recreate a new Xcode project for the generated code everytime because we need to modify the generated XCode project before we can add to AutorestswiftTest xcode project. And i don't want the manual changed to be overwritten everytime we run this script. The modification we need to make in the generated Xcode project is to remove the AzureCore target in each of the generated Xcode project. Without this modification, Xcode will complain saying there are multiple AzureCore targets (since each Xcode project will have its own AxureCore target by default (without the manually deletion)